.circleci/config.yml

```
@@ -259,56 +254,10 @@ workflows:
 
   patch_release:
     jobs:
-      - build:
-          filters:
-            branches:
-              only: /\d+\.\d+\.x$/
-
-      - default_test_job:
-          requires:
-            - build
-          prefixTestTask: true
-          name: test_<< matrix.testTask >>
-          matrix:
-            parameters:
-              testTask: ["7", "8", "14"]
-          filters:
-            branches:
-              only: /\d+\.\d+\.x$/
-
-      - default_test_job:
-          requires:
-            - build
-          name: test_11
-          testTask: test jacocoTestReport jacocoTestCoverageVerification
-          filters:
-            branches:
-              only: /\d+\.\d+\.x$/
-
-      - default_test_job:
-          requires:
-            - build
-          name: test_latest
-          testTask: latestDepTest
-          filters:
-            branches:
-              only: /\d+\.\d+\.x$/
-
-      - muzzle:
-          requires:
-            - build
-          filters:
-            branches:
-              only: /\d+\.\d+\.x$/
-
       - patch_release:
-          requires:
-            - test_7
-            - test_8
-            - test_11
-            - test_14
-            - test_latest
-            - muzzle
           filters:
             branches:
-              only: /\d+\.\d+\.x$/
+              ignore: /.*/
+            tags:
+              # Only for major and minor releases
+              only: /^v.*\.[1-9]+$/
```

comment update, and I really hope we never have 10 patch releases for one version, but just in case 😭
```suggestion
              # Only for patch releases
              only: /^v.*\.[1-9][0-9]*$/
```